package.json

```diff
@@ -26,14 +28,19 @@
   },
   "devDependencies": {
     "@ember/optional-features": "^0.6.3",
+    "@storybook/addon-actions": "^5.0.11",
+    "@storybook/addon-console": "^1.1.0",
+    "@storybook/ember": "^5.0.11",
+    "babel-loader": "^8.0.6",
     "broccoli-less-single": "^1.0.1",
     "ember-cli": "~3.4.3",
     "ember-cli-dependency-checker": "^3.0.0",
     "ember-cli-eslint": "^4.2.3",
     "ember-cli-htmlbars": "^3.0.0",
-    "ember-cli-htmlbars-inline-precompile": "^1.0.3",
+    "ember-cli-htmlbars-inline-precompile": "^2.1.0",
     "ember-cli-inject-live-reload": "^1.8.2",
     "ember-cli-qunit": "^4.3.2",
+    "ember-cli-storybook": "^0.1.0",
```

Comment:
I don't think `ember-cli-storybook` is necessary. The docs seem to indicate that we just need `@storybook/ember` everything seemed to work just fine when I removed locally